urdf_usd_converter/_impl/geometry.py

```diff
@@ -0,0 +1,70 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 The Newton Developers
+# SPDX-License-Identifier: Apache-2.0
+import usdex.core
+from pxr import Gf, Tf, Usd, UsdGeom
+
+from .data import ConversionData, Tokens
+from .urdf_parser.elements import (
+    ElementBox,
+    ElementCylinder,
+    ElementGeometry,
+    ElementMesh,
+    ElementSphere,
+)
+from .utils import float3_to_vec3d
+
+__all__ = ["convert_geometry"]
+
+
+def convert_geometry(parent: Usd.Prim, name: str, geometry: ElementGeometry, data: ConversionData) -> UsdGeom.Gprim:
+    if isinstance(geometry.shape, ElementBox):
+        return convert_box(parent, name, geometry.shape, data)
+    elif isinstance(geometry.shape, ElementSphere):
+        return convert_sphere(parent, name, geometry.shape, data)
+    elif isinstance(geometry.shape, ElementCylinder):
+        return convert_cylinder(parent, name, geometry.shape, data)
+    elif isinstance(geometry.shape, ElementMesh):
+        return convert_mesh(parent, name, geometry.shape, data)
+
+
+def convert_box(parent: Usd.Prim, name: str, box: ElementBox, data: ConversionData) -> UsdGeom.Gprim:
+    cube_prim = usdex.core.defineCube(parent, name, 1.0)
+    size = float3_to_vec3d(box.get_with_default("size"))
+    usdex.core.setLocalTransform(cube_prim.GetPrim(), Gf.Vec3d(0), Gf.Quatf.GetIdentity(), Gf.Vec3f(size))
```

Comment:
I think it would be better to just add a scaleop as we do in mjc https://github.com/newton-physics/mujoco-usd-converter/blob/main/mujoco_usd_converter/_impl/geom.py#L178-L180